scss/_patterns_tabs.scss

```diff
@@ -46,7 +46,7 @@
       border: none;
       color: $color-dark;
       display: flex;
-      gap: $spv--medium;
```

Comment:
There is no "medium" in horizontal spacing, use `$sph--small` or `$sph-large` in this case (I guess small should be enough?)